src/components/callout/callout.e2e.ts

```diff
@@ -0,0 +1,49 @@
+import { newE2EPage } from '@stencil/core/testing';
```

Comment:
Nice! Seems like we have this in a pretty good place now 👍

The one thing I would like to change before we merge this, is that I would like to limit the number of supported types.

It's super-easy to just add more supported types in the future, but once added, removing one is a breaking change, and once it's widely used by Lime CRM add-ons, we basically can't ever remove it.

I'm worried that this many types will be more confusing than helpful, and I'm not sure all of the current types are well thought through. Why is `tip` green, but `success` blue? Do we _really_ need a `danger` when we have `warning`, and if we do, why is it the second most muted colour, after only `quote`? Should we really have a `quote` type when the semantic element to use for block quotes is `<blockquote>`?

The following are the ones supported by default by the plugin we use in Kompendium. I don't _love_ them, but at least it's a limited set. Could we go for something a little closer to those five, rather than our current _ten_?

![image](https://user-images.githubusercontent.com/154725/228813641-3dd17ab9-762a-4b36-b28f-3cd1a00fc096.png)

Reply:
If you're OK with this @Kiarokh and @TommyLindh2, I can make the change.

Reply:
It doesn't matter at all for me, please do what you guys think is best.